spec/defines/collectd_plugin_curl_json_spec.rb

```diff
@@ -13,6 +13,9 @@
     {
       url: 'http://localhost:55672/api/overview',
       instance: 'rabbitmq_overview',
+      verifypeer: 'false',
+      verifyhost: 'false',
```

Comment:
I am not sure if 'false' here equals the default undef in the manifest.

CC: @jyaworski